docs/examples/translation_to_specific_language.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Add profanity check on the translated text\n",
+    "\n",
+    "!!! note\n",
+    "    To download this example as a Jupyter notebook, click [here](https://github.com/ShreyaR/guardrails/blob/main/docs/examples/translation_to_specific_language.ipynb).\n",
+    "\n",
+    "In this example, we will use Guardrails during the translation of a statement from another language to english. We will check whether the translated statement passes the profanity check or not.\n",
+    "\n",
+    "## Objective\n",
+    "\n",
+    "We want to translate a statement from another languages to English and ensure the translated statement is profanity free.\n",
+    "\n",
+    "## Step 0: Setup\n",
+    "\n",
+    "In order to run this example, you will need to install both `alt-profanity-check` and `guardrails-ai` packages. You can do so by running the following commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install alt-profanity-check\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install guardrails-ai"
```

Comment:
nit: For consistency with other examples, can you remove `guardrails-ai` installation command?

Reply:
done